config/manager/manager.yaml

```diff
@@ -70,6 +72,17 @@ spec:
             name: vault-approle
             readOnly: true
             subPath: role_id
+      - name: kube-rbac-proxy
+        image: gcr.io/kubebuilder/kube-rbac-proxy:v0.8.0
+        args:
+          - "--secure-listen-address=0.0.0.0:8443"
+          - "--upstream=http://127.0.0.1:8080/"
+          - "--logtostderr=true"
+          - "--v=10"
```

Comment:
How verbose is this?

Reply:
K, let's keep it for a little while, but let's revisit this in the next release or something..